### ISSUE TITLE
How can I reach the result for Resnext101 on Hmdb51

### ISSUE THREAD

**Issue by BarakBat** [kenshohara/3D-ResNets-PyTorch#128]
Hi I'm trying to reach the score that was written in the paper with Resnext3d101 on Hmdb51.
I'm taking your pretrained model (on kinetics), taking down the last fc :(2048,400), putting new fc:(2048,51).
freeze the backcone, and trianing on HMDB51, But I'm reaching aroung 0.43 instead of 0.71.
All training Hyper Parameters are the same as the paper.
Any tip about how to get this result?!?

**Comment by xuyu0010** [comment]
Hi, I am also getting poorer results on HMDB51 dataset, any clues on why the accuracy is lower? Thanks!
